packages/autorest.python/README.md

```diff
@@ -122,26 +117,6 @@ scope-multiapiscript/emitter:
 output-artifact: python-files
 ```
 
-# Black script pipeline
-
-```yaml $(black)
-pipeline:
-  python/black:
-    scope: black
-    input: python/codegen
-    output-artifact: python-files
-
-  python/black/emitter:
-    input: black
-    scope: scope-black/emitter
-
-scope-black/emitter:
-  input-artifact: python-files
-  output-uri-expr: $key
-
-output-artifact: python-files
-```
-
```

Comment:
I don't find effective way to flush the buffer into disk. But if we format the file content before write, we can avoid the annoying problem. What's more, we reduce black plugin to make the flow shorter which is  cleaner.

Reply:
`and it would be breaking to make users not be able to turn off black`
users can still turn off black: https://github.com/Azure/autorest.python/pull/1410/files#r961250499 and https://github.com/Azure/autorest.python/pull/1410/files#r961250788 which will not bring breaking change